docker-compose/oidc/README.adoc

```diff
@@ -0,0 +1,361 @@
+= Enable Unified Identity with Azure Entra ID for Redpanda and Redpanda Console
+:env-docker: true
+:page-categories: Security, Management, Redpanda Console
+:description: Integrate Azure Entra ID with Redpanda and Redpanda Console for unified identity using OpenID Connect (OIDC).
+:page-layout: lab
+// Set up attributes to hold the latest version of Redpanda and Redpanda Console.
+// For GitHub, hard-code the latest version to these values:
+ifndef::env-site[]
+:latest-redpanda-version: 25.1.1
+:latest-console-version: 3.1.0
+endif::[]
+// For the docs site, use the built-in attributes that store the latest version as fetched from GitHub releases.
+ifdef::env-site[]
+:latest-redpanda-version: {full-version}
+// All pages already have access to {latest-console-version} on the docs site.
+endif::[]
+
+This lab walks you through integrating Azure Entra ID (formerly Azure AD) with Redpanda and Redpanda Console for unified identity using OpenID Connect (OIDC). This integration allows you to use Azure Entra ID for authentication and authorization in Redpanda and Redpanda Console.
```

Comment:
I'm not sure I would say "authorization" from Entra ID is possible since users cannot map a groups token claim to a Redpanda role. So, they can't effectively control authorization from Entra ID, only authentication.